pkg/apis/core/types.go

```diff
@@ -2189,37 +2189,31 @@ type Container struct {
 }
 
 // ProbeHandler defines a specific action that should be taken in a probe.
-// This type has a strong relationship to LifecycleHandler - overlapping fields
-// should be identical.
-// TODO: pass structured data to these actions, and document that data here.
+// One and only one of the following should be specified.
```

Comment:
moved this text here as otherwise it become a comment of an Exec probe only.

Reply:
Maybe "One and only one field should be specified." ?

Reply:
Specifically "of the following" is weird outside the struct